src/commands/project/create-variable-group.ts

```diff
@@ -297,8 +317,12 @@ export const execute = async (
     );
     await exitFn(0);
   } catch (err) {
-    logger.error(`Error occurred while creating variable group`);
-    logger.error(err);
+    logError(
+      buildError(
+        errorStatusCode.CMD_EXE_ERR,
+        "project-create-variable-group-cmd-failed"
+      )
```

Comment:
```
      buildError(
        errorStatusCode.CMD_EXE_ERR,
        "project-create-variable-group-cmd-failed",
        err
      )
```
important to add err. otherwise the error chain will not be formed